.github/workflows/tests.yml

```diff
@@ -0,0 +1,23 @@
+name: CI
+
+on: [push, pull_request]
```

Comment:
I would keep the tests running only on `pull_request` on `develop` branch

Reply:
Done @RaduCStefanescu